### PR TITLE
Added Headless browser support to existing list of browser

### PR DESCRIPTION
**Pull request by atiqsayyed**
simply run "sbt run compare headless" to run tests in headless browser

If you have any problem related to phantomjs, please look at http://stackoverflow.com/questions/28267809/phantomjs-getting-killed-9-for-anything-im-trying and try downloading relevant phantomjs distribution from here https://github.com/eugene1g/phantomjs/releases

### CODE REVIEW

**Comment by hrj** [comment]
Saw this PR just now! Good work @atiqsayyed 

Can you see if they provide a common interface that a `gngr` specific driver can inherit? We can then avoid the selenium dependency entirely.

Otherwise, I am a little hesitant to pull this in because it will increase the number of dependencies. Though phantomjs is faster than selenium+firefox, we rarely require to run firefox / phantomjs. We will be running `gngr` most of the time.

File: report.html [hunk 1]
```diff
@@ -53,15 +53,21 @@
 
 <script>
   function formSubmitted(f) {
+<<<<<<< HEAD
+=======
```
Comment (hrj) [inline report.html]: These are merge conflicts. You need to delete the lines that begin with `<<<<`, `====` and `>>>>`

**Comment by atiqsayyed** [comment]
@hrj even i felt so. . . PhantomJs isn't much value add. . 
will play with the PhantomJs driver to see if we can use it somehow. . .
not sure as of now. .  but will try to find something. . 

We can ignore this PR as of now. . 

PS: I'm very lazy when it comes to paying attention while commiting. .  You might have experienced this by now ;D

**Comment by hrj** [comment]
Thanks @atiqsayyed 

As discussed, going to close this PR for now.

**Pull request closed without merging**